spec/takeaway_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { Takeaway.new }
```

Comment:
# Appreciate how you've defined takeaway instead of subject. It's easier to follow